kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -28,9 +32,7 @@
 import java.util.Map;
 import java.util.Properties;
 
-import io.confluent.common.config.ConfigDef;
-import io.confluent.common.config.ConfigDef.Importance;
-import io.confluent.common.config.ConfigDef.Type;
+import io.confluent.common.config.AbstractConfig; // TODO: remove this
```

Comment:
this is used for `getPropsFromFile`. is the `TODO` realistic or is that utility going to be used indefinitely? (I'm not sure why we care about removing this...)

Reply:
Why not just drop this import and inline the `getPropsFromFile` method, if that's all its needed for?  Again, `getPropsFromFile` could be in some util class...

Reply:
`getPropsFromFile` is used throughout cp codebase. i left this todo here to remind me to do exactly what you all are proposing.

i was trying to not tie together all the changes at once. i can do that if you all think it would make these changes easier

Reply:
maybe thats not worth it...